pkg/daemon/handler_test.go

```diff
@@ -86,7 +87,8 @@ func TestHandler(t *testing.T) {
 					Authenticated: true,
 					User: authv1.UserInfo{
 						Extra: map[string]authv1.ExtraValue{
-							"authentication.kubernetes.io/pod-uid": {"the-pod-name"},
+							"authentication.kubernetes.io/pod-uid":  {"the-pod-name"},
+							"authentication.kubernetes.io/pod-name": {"the-pod-name"},
```

Comment:
if these are the same it doesn't really test that we used the right field

Reply:
actually, I don't even this this update was needed (we don't use `authentication.kubernetes.io/pod-name` anywhere in the test). But I'll update the `pod-uid` value to `the-pod-uid` for clarity